frontend/components/ManageStakingPage/StakingContractSection/CantMigrateAlert.tsx

```diff
@@ -86,12 +86,13 @@ const AlertInsufficientMigrationFunds = ({
             <ul style={{ marginTop: 0, marginBottom: 4 }}>
               {requiredOlasDeposit > 0 && (
                 <li>
-                  {requiredOlasDeposit} {TokenSymbol.OLAS}
+                  {requiredOlasDeposit.toLocaleString()} {TokenSymbol.OLAS}
```

Comment:
tiny: would prefer to use `balanceFormat` for consistency cross-platform, https://github.com/valory-xyz/olas-operate-app/blob/fe9d88970c620c9753c43ba7b47b6f26847b14f8/frontend/utils/numberFormatters.ts#L6